hack/package_chart.sh

```
@@ -0,0 +1,56 @@
+
+rm -rf "$output_dir"
+mkdir -p "$output_dir"
+cat $input_dir/versions_map | while read package version commit
+do
+        git archive --format tar $commit $input_dir/$package | tar -xf- -C $tmp_dir/
+        if [ "x$dependency_dir" != "x" ] && git ls-tree --name-only $commit $dependency_dir | grep -qx $dependency_dir
+        then
+                git archive --format tar $commit $dependency_dir | tar -xf- -C $tmp_dir/
+        fi
+        helm package $tmp_dir/$input_dir/$package -d $output_dir
+        rm -rf $tmp_dir/$input_dir/$package
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)